src/release/options.d

```diff
@@ -113,3 +113,49 @@ Options parseOpts ( string[] opts )
 
     return options;
 }
+
+
+/*******************************************************************************
+
+    Note: This doesn't strictly belong here, but it can't live in the main.d
+    module because that one isn't unittested. Feel free to move if a better
+    place comes to mind.
+
+    Simple and stupid domain extractor. Looks for the string between @ and : and
+    returns it. Throws an exception if either symbol wasn't found
+
+    Params:
+        url = url to find
+
+    Returns:
+        domain part of the url
+
+*******************************************************************************/
+
+public string extractDomain ( string url )
+{
+    import std.algorithm;
+    import std.range : empty;
+
+    auto begin = url.findSplitAfter("@");
+
+    if (!begin)
```

Comment:
You don't like `enforce`, do you? :)

Reply:
It's just not a pattern I am used too :-)

Reply:
It's just not a pattern I am used too :-)

Reply:
It's just not a pattern I am used too :-)